src/ophys_etl/modules/maximum_projection/__main__.py

```diff
@@ -0,0 +1,113 @@
+import h5py
+import PIL.Image
+import argschema
+from marshmallow import post_load
+
+from ophys_etl.modules.maximum_projection.utils import (
+    maximum_projection_from_path,
+    scale_to_uint8)
+
+
+class MaximumProjectionSchema(argschema.ArgSchema):
```

Comment:
To avoid future confusion it might be good to call this whole module and the output image a `median filtered max projection`. `FilteredMaximumProjectionSchema`

Reply:
Good point. Especially since we haven't yet figured out whether we actually want to do this automatically or when (i.e. as a part of motion correction or as a stand alone module).